SR2021/2021-02-14-league-2-followup-no-shows.md

```diff
@@ -0,0 +1,31 @@
+---
+to: Teams who didn't attend the leagues
+subject:
+    - Still interested in Student Robotics 2021?  # if they appear inactive
+    - Would you like a mentor?  # if they appear active
+---
+
+Hi,
+
+We noticed your team didn't make it to the recent league sessions. We wanted to
+reach out to see if there was any more support we could provide you, or questions
+we could answer?
+
+There's usually lots of volunteers around in Discord to help out if you need
+anything, whether it's help programming your robot, organising your team or you
+just need a fresh pair of eyes on something. Our volunteers have a wealth of
+experience from both competing in Student Robotics and helping lots of teams, so
+there's a good chance they can help :)
+
+We'd also like to offer you some more hands-on support in the form of a regular
+mentor. This would be a volunteer who is able to be around whenever you
+regularly meet and would get to know your team and your robot. Whatever platform
+you use (whether it's Discord or something else), they'd be able to join your
+meetings to help you out. (For safeguarding reasons we need your team leader to
+also be in any video calls; your team leader will have details about this).
+
```

Comment:
I think it's also worth mentioning if they don't wish to continue?

Reply:
Ah, I'd meant to include a comment about this in the commit message. I thought about this but thought that if they were on the fence then mentioning it might encourage them to do so. Teams who have dropped out will probably let us know anyway I'd hope. On the other hand maybe at this point we think they're likely gone enough that they're not coming back, so encouraging the clarity might be preferable?

Reply:
Yeah no signs of life for 4 months aren't great signs. I think it's worth knowing and suggesting.

Reply:
I realise we don't have any actual templating here, though hopefully fe2ec9dea18f634168c65caff591e1234799c6a1 is clear enough.